### ISSUE TITLE
Ability to resize (enlarge) cluster icons for readability

### ISSUE THREAD

**Issue by ten-chs** [lensapp/lens#3645]
**What would you like to be added**:
Lens currently has no way to change the size of icons anywhere in the app (that I can find). I'm specifically requesting a way to change (increase) the size of the icons for clusters for readability.

**Why is this needed**:
It seems like basic functionality that is missing. Also, when using many clusters, ideas for distinct and easily distinguishable cluster icons can be hard to achieve. Our current implementation is the acronym for the specific cluster inside a circle, whose color is determined by the level of the environment (staging, dev, pre-prod, prod, post-prod, etc.)

**Environment you are Lens application on:**

- Kubernetes distribution: EKS
- Desktop OS: MacOs

**Comment by Nokel81** [comment]
These icon sizes are all placement dependent. Namely I can think of at least three different sizes we have currently: the hotbar icons, the icons in the catalog, and the catalog entity details panel.

So I guess I am a bit confused as to what you are asking, you can already force a specific colour for those icons and I think we are planning on adding the ability to specify a "short name" which would be used instead

**Comment by ten-chs** [comment]
I'm proposing being able to change the size of the icons within whichever view you're in, rather than only having them be the default size there.

**Comment by Nokel81** [comment]
> It seems like basic functionality that is missing. 

Would you be able to share some examples? Favicons are not generally resizable in browser tabs for instance.

Would you be able to provide a screenshot so that we can better understand the issue you are having with intelligibility?

Finally, it seems that you have different environments, would having each of the clusters of each environment on a different hotbar (which can be named to the environment names) help?

**Comment by ten-chs** [comment]
Work would not approve of images of our dashboards being publicly released, I'm sure you understand.

Suffice it to say, the Clusters tab/pane within the catalog could use the option to resize the icons. At their static size, they are almost undistinguishable from one another, aside from the coloring (only distinguishes groups, not individuals). Similarly, the name for each is so long, that its distinctness isn't visible in the current view.

I have to click into each one in the group (again by coloring in the icon) to figure out which one I actually want to monitor.

**Comment by Nokel81** [comment]
Fair enough, you can currently provide an alternative name for clusters and would several hotbars help?

**Comment by ten-chs** [comment]
I didn't realize I could provide an alternative name. How is this accomplished? I assumed the connection required the current formatting.
For multiple hotbars, yes, I believe we will get to that requirement, if we haven't already.

**Comment by Nokel81** [comment]
You can specify an alternative name under cluster settings, it is purely a visual change within Lens.

**Comment by blevine** [comment]
> ... I think we are planning on adding the ability to specify a "short name" which would be used instead

@Nokel81  I would absolutely love this! My cluster naming convention is such that the 3-letter abbreviation in the cluster icon is sometimes repeated.  Yes, I can hover over the icon and get the full name, but it would be nice to be able to easily tell the difference (and use my own labels) at a glance.

**Comment by Nokel81** [comment]
@blevine Please go upvote #3498
